app/controllers/favorites_controller.rb

```diff
@@ -9,7 +9,7 @@ def update
   end
 
   def index
-    @pets = Pet.find(favorites.contents)
+    @pets = Pet.where(id: favorites.contents)
```

Comment:
should make it so if someone else deletes a pet that's in your favorites, the page can still render